core/src/gl/shaderProgram.cpp

```diff
@@ -84,15 +120,20 @@ GLint ShaderProgram::getAttribLocation(const std::string& _attribName) {
 
 }
 
-GLint ShaderProgram::getUniformLocation(const std::string& _uniformName) {
+GLint ShaderProgram::getUniformLocation(const UniformEntries::UniformEntry* _entry) {
```

Comment:
Right, I preferred the const pointer to make it clear that this can be null, but since a uniform ID of 0 is invalid that's also a good solution, I'll update for this.